scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -462,12 +462,14 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 }
             }
 
+            let minTansaUrgency = $rootScope.config.tansa && $rootScope.config.tansa.urgency || 5;
```

Comment:
Another option is for 5 to live in the defaults, over at the default [config](https://github.com/superdesk/superdesk-client-core/blob/master/webpack.config.js#L119). Not sure if that's a better option, but it would completely remove this line and you could use `$rootScope.config.tansa.urgency` directly.